api/v1beta1/ovnnorthd_types.go

```diff
@@ -75,6 +75,11 @@ type OVNNorthdSpecCore struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// TLS - Parameters related to TLS
 	TLS tls.SimpleService `json:"tls,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=1
+	// NThreads sets number of threads used for building logical flows
```

Comment:
nThreads? Or maybe also consider renaming the field into numThreads so that it's more clear what this is about.

Reply:
My problem is with the comment, because this comment gets into CRD yamls (which is documentation) where it says that the name of the API field starts with upper case N (but in reality it's nThreads, as in your json: metadata).

Reply:
I agree, but again that is not what I see. Other variables like the `containerImage, loglevel, networkAttachments` use lower case letter for the json key, upper for the comment description and upper for the variable name in the source code. I simply just followed the established form. 
If the requirement is to have variable in the upper case then we can follow up with another PR where we make this change accross everything.

Reply:
I see your point. Yes, I think others shouldn't list names of fields in wrong caps too. But I don't want to block this patch for this.